services/server/src/server/services/storageServices/AbstractDatabaseService.ts

```diff
@@ -16,12 +16,10 @@ export default abstract class AbstractDatabaseService {
     return await this.database.initDatabasePool(this.IDENTIFIER);
   }
 
-  validateVerificationBeforeStoring(verification: Verification): boolean {
+  validateVerificationBeforeStoring(verification: VerificationExport): boolean {
     if (
       verification.compilation.runtimeBytecode === undefined ||
-      verification.compilation.creationBytecode === undefined ||
-      verification.status.runtimeMatch === undefined ||
-      verification.status.creationMatch === undefined
+      verification.compilation.creationBytecode === undefined
```

Comment:
No, here we want to store only if we have creation and runtime match

Reply:
because this is the generic `validateVerificationBeforeStoring` used by the VerA database